lib/common/windowController.py

```diff
@@ -0,0 +1,75 @@
+import time
+import sys
+import subprocess
+
+if sys.platform == "win32":
+   import win32gui
+   import win32con
```

Comment:
For Windows platform, if we can raise the error message "`please download and install pywin32 from https://sourceforge.net/projects/pywin32/files/pywin32/`" when import modules failed (origin message will be "`ImportError: No module named win32con`"), I think it will be more helpful.

ex:
```
try:
    import win32gui
    import win32con
except ImportError as e:
    print(e)
    raise ImportError('Please download and install pywin32 from https://sourceforge.net/projects/pywin32/files/pywin32/')